services/spar/test-integration/Util/Scim.hs

```diff
@@ -623,5 +620,21 @@ userShouldMatch u1 u2 = liftIO $ do
 -- floor.  This function calls the spar functions that do that.  This allows us to express
 -- what we expect a user that comes back from spar to look like in terms of what it looked
 -- like when we sent it there.
-whatSparReturnsFor :: HasCallStack => IdP -> Int -> Scim.User.User SparTag -> Either String (Scim.User.User SparTag)
-whatSparReturnsFor idp richInfoSizeLimit = either (Left . show) (Right . synthesizeScimUser) . validateScimUser' (Just idp) richInfoSizeLimit
+whatSparReturnsFor :: HasCallStack => TeamId -> IdP -> Int -> Scim.User.User SparTag -> Either String (Scim.User.User SparTag)
+whatSparReturnsFor tid idp richInfoSizeLimit =
+  either (Left . show) (Right . synthesizeScimUser)
+    . validateScimUser' tid (Just idp) richInfoSizeLimit
+
+renderAuthIdAsExternalId :: AuthId -> Maybe Text
```

Comment:
move this to wire-api?  (you can refuse, or do it in the next PR.)

Reply:
I thought about this, but since this function is only used in the integration tests I didn't want to elevate its officialness.